### PR TITLE
Prepare 3.45.0

### PR DESCRIPTION
**Pull request by jstvz**
-------

# Critical Changes

# Changes

# Issues Closed

### CODE REVIEW

File: HISTORY.rst [hunk 1]
```diff
@@ -2,6 +2,26 @@
 History
 =======
 
+v3.45.0 (2021-09-30)
+--------------------
+
+Critical Changes
+
+-  Updated the ``deploy_marketing_cloud_package`` task for compatibility with the October 2021 release of Marketing Cloud. (#2899)
```
Comment (davisagli) [inline HISTORY.rst:10]: this should be under changes, not critical changes. the only change was adding support for a new return status, so it's backwards compatible

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,26 @@
...
+
+Changes
+
+-  Fixed a bug in freezing the ``load_dataset`` task options for MetaDeploy. (#2900)
```
Comment (davisagli) [inline HISTORY.rst:14]: should go under issues fixed

File: HISTORY.rst [hunk 3]
```diff
@@ -2,6 +2,26 @@
...
+-  The marketing cloud ``deploy`` task now properly exits when a result status of ``FATAL_ERROR`` is returned. (#2897)
```
Comment (davisagli) [inline HISTORY.rst:15]: should go under issues fixed

File: HISTORY.rst [hunk 4]
```diff
@@ -2,6 +2,26 @@
...
+-  The ``--max-lines`` option on the ``cci error info`` command has been removed. (#2895)
+-  We changed how some the output from some commands and tasks are displayed in the CLI. (#2887)
```
Comment (davisagli) [inline HISTORY.rst:17]:
```suggestion
-  We changed how the output from some commands and tasks are displayed in the CLI. (#2887)
```

**Review by davisagli — CHANGES_REQUESTED** [review]

**Review by davisagli — APPROVED** [review]

File: HISTORY.rst [hunk 5]
```diff
@@ -2,6 +2,23 @@
...
+Changes
+
+-  Updated the ``deploy_marketing_cloud_package`` task for compatibility with the October 2021 release of Marketing Cloud. (#2899)
+-  The ``--max-lines`` option on the ``cci error info`` command has been removed. (#2895)
+
+Issues fixed
+
+-  Removed the unused ``--skip`` option for the ``cci flow run`` command. (#2884)
+-  Flow descriptions no longer generate a warning. (#2885)
+-  We changed how the output from some commands and tasks are displayed in the CLI. (#2887)
```
Comment (davisagli) [inline HISTORY.rst:17]: this seems more like a change than an issue fixed, but not worth holding up the release for

Comment (jstvz) [reply HISTORY.rst:17]: Will tweak in the release body.

**Pull request merged**